Review link content

grange85 opened the pull request:
Hi Gwyn, I've added links and terms so that recommended links are shown when users search for 'log in', 'sign in' or 'account' - could you check this over and let me know if anything's awry.

If all's good can we look at deploying it to search.

Thanks

grange85 committed:
Update broken and redirected links

grange85 committed:
Add links for Discovery, bookshop and img lib

grange85 committed:
Add terms for sign in log in and account

gtvj reviewed (approved):
Hi @grange85 - looks good to me. Thank you. I'll add the files to the project.

gtvj committed:
Add minified JavaScript files

Files compiled by GruntJS. All QUnit tests passing, including those
testing the newly added terms/regex
